src/components/bulkConcepts/addBulkConcepts.jsx

```diff
@@ -82,11 +85,18 @@ export class AddBulkConcepts extends Component {
     const lastPage = Math.ceil(cielConcepts.length / conceptsPerPage);
     const lastConcept = indexOfFirstConcept + currentConcepts.length;
     const firstConcept = indexOfLastConcept - 19;
+
+    const disableButton = ((this.state.cielConcepts.length === 0) || this.props.isFetching);
     return (
       <div className="container-fluid add-bulk-concepts custom-max-width">
         <Header locationPath={this.props.match.params} />
         <h3>
-          <strong>{dictionaryName} Dictionary</strong>: Bulk Add Concepts
+          <strong>
+            {dictionaryName}
+            {' '}
+            Dictionary
+          </strong>
+          : Bulk Add Concepts
```

Comment:
@ozezzy , all changes that appear based on linting should not be in this ticket.

Reply:
Thanks for pointing that out @Duncanian. I have updated this to remove all changes done by the linter